book/src/keymap.md

```diff
@@ -67,6 +67,8 @@
 | `=`         | Format selection                                | `format_selections`   |
 | `d`         | Delete selection                                | `delete_selection`    |
 | `c`         | Change selection (delete and enter insert mode) | `change_selection`    |
+| `Ctrl-a`    | Increment object under cursor                   | `increment`           |
+| `Ctrl-x`    | Decrement object under cursor                   | `decrement`           |
```

Comment:
```suggestion
| `Ctrl-a`    | Increment object (number) under cursor          | `increment`           |
| `Ctrl-x`    | Decrement object (number) under cursor          | `decrement`           |
```
Make the docs more specific.

Reply:
I removed "number" from these because of an earlier comment where you said you didn't want these to be limited to numbers, but I can definitely add it back.